src/lib/authenticator.ts

```diff
@@ -355,12 +357,11 @@ export class GenesysCloudClientAuthenticator {
       this._debug('Implicit grant: opening new window: ' + loginUrl);
 
       /* this will always be `null` if `nofererrer` or `noopener` is set */
-      /* const popupWindow =  */window.open(loginUrl, '_blank', 'width=500px,height=500px,noreferrer,noopener,resizable,scrollbars,status') as Window;
+      window.open(loginUrl, '_blank', 'width=500px,height=500px,noreferrer,noopener,resizable,scrollbars,status') as Window;
 
       const timeoutId = setTimeout(() => {
         this._debug('timeout for loginImplicitGrant using popup', query);
-        const error = new Error('Popup timeout. It is possible that the popup was blocked.');
-        error.name = 'POPUP_TIMEOUT';
+        const error = new TimeoutError('Popup authentation timeout. It is possible that the popup was blocked or the login page encountered an error');
```

Comment:
This is the breaking change. I decided to switch it to `TIMEOUT_ERROR` instead of `POPUP_TIMEOUT` because we don't actually know if it was a popup timeout or some other auth timeout.